lib/segment/src/index/hnsw_index/graph_layers.rs

```diff
@@ -280,12 +293,21 @@ where
     }
 
     pub fn save(&self, path: &Path) -> OperationResult<()> {
```

Comment:
Not related to this PR, but I think we better rename this method to something like `save_params` to avoid confusion